gradle/libs.versions.toml

```diff
@@ -131,6 +131,7 @@ spotless = "com.diffplug.spotless:6.4.2"
 toml-checker = "com.github.ben-manes.versions:0.42.0"
 toml-updater = "nl.littlerobots.version-catalog-update:0.3.1"
 ksp = { id = "com.google.devtools.ksp", version.ref = "ksp" }
+jlleitschuh = "org.jlleitschuh.gradle.ktlint: 11.6.1"
```

Comment:
try something like this.
`jlleitschuh = {"org.jlleitschuh.gradle.ktlint", version.ref = "ktlint"}`
and then add ktlint at the top of the file 
`ktlint = 11.6.1`

Reply:
yes sure.

Reply:
I've done so